Gems/PhysX/Code/Tests/CharacterGameplayControllerTests.cpp

```diff
@@ -151,7 +154,7 @@ namespace PhysX
         
         // Let scene run for a few moments so the entity can be manipulated by gravity from the gameplay component
         const auto startPosition = basis.m_controller->GetPosition();
-        const int timeStepCount = rand() % 180 + 10;
+        const int timeStepCount = GetParam(); // <-- Gets the current value to use
```

Comment:
nit: Please remove the comment, I put them in the feedback to remark the changes for you :)

Reply:
Removed